src/test/java/com/google/cloud/spanner/jdbc/it/ITJdbcSimpleStatementsTest.java

```diff
@@ -59,14 +65,24 @@ public static List<DialectTestParameter> data() {
 
   @Parameter public DialectTestParameter dialect;
 
+  private Database database;
+
+  @Before
+  public void setup() {
+    assumeFalse(
+        "Emulator does not support PostgreSQL",
+        dialect.dialect == Dialect.POSTGRESQL && EmulatorSpannerHelper.isUsingEmulator());
+    database = env.getOrCreateDatabase(getDialect(), Collections.emptyList());
+  }
+
   @Override
   public Dialect getDialect() {
```

Comment:
should we make `getDialect ` private? `setup` method seems to be taking care of things now